src/sagas/wallet.js

```diff
@@ -543,11 +530,11 @@ export function* saga() {
   yield all([
     takeLatest('START_WALLET_REQUESTED', startWallet),
     takeLatest('WALLET_CONN_STATE_UPDATE', onWalletConnStateUpdate),
-    takeLatest('WALLET_RELOADING', walletReloading),
+    takeLatest('WALLET_RELOADING', onWalletReloadData),
+    takeLatest('WALLET_RELOAD_DATA', onWalletReloadData),
```

Comment:
Didn't we remove this action?

Reply:
Fixed in ca011434a77409fb5a8b22e73f8c3f66dd03e7e6